Mappings 2.0 [DOC-990] (#7006)
segmentio/segment-docs: src/connections/destinations/index.md

In order to override the default, check the destination settings pane in the Segment web App either for a **Connection Mode** toggle or instructions on bundling any additional mobile components required.

## Sync modes

> Segment released a new sync mode capability that allows users to define how changes in the source should send downstream to your destination. Depending on which destinations you set up in Segment, you may need to choose a sync mode for your data. This configuration determines how Segment updates your destination based on the source data. 

Reviewer: ```suggestion
Sync mode capability allows users to define how changes in the source should send downstream to your destination. Depending on which destinations you set up in Segment, you may need to choose a sync mode for your data. This configuration determines how Segment updates your destination based on the source data. 
```

In order to override the default, check the destination settings pane in the Segment web App either for a **Connection Mode** toggle or instructions on bundling any additional mobile components required.

## Sync modes

> In September 2024, Segment released a new sync modes capability that allows users to define how changes in the source should be sent downstream to your destination. Depending on which destinations you set up in Segment, you may need to choose a sync mode for your data. This configuration determines how Segment updates destination based on the source data. 

Reviewer: ```suggestion
In September 2024, Segment released a new sync mode capability that allows users to define how changes in the source should send downstream to your destination. Depending on which destinations you set up in Segment, you may need to choose a sync mode for your data. This configuration determines how Segment updates your destination based on the source data. 
```

Reviewer: Do we need to say "In September 2024, Segment released..."? Just wondering because for every new feature or update, we don't say when it came out.

Author: yeah, fair point - I did this to match with the Destination Actions section that's above it, but it's probably overkill

The available sync modes can vary based on the destination, integration type, and actions within the destination. For example, if you sync customer data, you might have the option to Insert, Update, or Upsert records.

Available sync modes include: 
- **Update**: Modify existing records in the destination without adding new ones.

> - **Insert**: Add new records without altering existing data.

Author: ```suggestion
```
